### ISSUE TITLE
Clarification regarding patch size?

### ISSUE THREAD

**Issue by kadarsh22** [amirbar/StoP#4]
I noticed a possible inconsistency between the paper and the public implementation regarding the ViT-H backbone configuration.

In the paper, the main reported results appear to use ViT-H/14. For example, the tables refer to StoP / I-JEPA results with a ViT-H/14 backbone.

However, in the released repo, the huge ViT pretraining config seems to use patch size 16:

configs/pretrain/vit-h16.yaml is named vit-h16
The config sets:
mask:
  patch_size: 16

Could you clarify which setup was used for the reported ViT-H results in the paper?

Specifically:

Were the paper’s ViT-H results trained/evaluated with patch size 14 or patch size 16?
Is the released vit-h16.yaml config intended to reproduce the paper’s ViT-H/14 results, or is it a separate ViT-H/16 release?

**Comment by amirbar** [comment]
Hi @kadarsh22, nice catch!
 
The paper's ViT-H results were trained and evaluated with patch size 14 (ViT-H/14), matching I-JEPA's setup

You can verify this directly from the released checkpoint: patch_embed.proj.weight has shape (1280, 3, 14, 14) and pos_embed has 256 positions.

The vit-h16.yaml filename and its mask.patch_size: 16 value are a mistake carried over from the ViT-B/16 and ViT-L/16 configs. Running that config would train an H/16 model and would not reproduce the paper numbers.

**Comment by amirbar** [comment]
fixed here:
https://github.com/amirbar/StoP/commit/bb385a29a49d123be19743a25849f76a346ab576
